packages/lab/stories/breakpoints.stories.tsx

```diff
@@ -0,0 +1,145 @@
+import { StackLayout, Button, Card } from "@salt-ds/core";
+import {
+  makeResponsive,
+  ResponsiveChildren,
+  BreakpointsProvider,
+  useResponsiveValue,
+  useResponsiveProps,
+  createResponsiveSystem,
+} from "@salt-ds/lab";
+
+export default {
+  title: "Lab/Responsive Props",
+};
+
+const ResponsiveStackLayout = makeResponsive(StackLayout);
+
+export const MakeResponsiveExample = () => (
+  <BreakpointsProvider>
+    <ResponsiveStackLayout
+      gap={{
+        default: 0,
+        sm: 1,
+        md: 2,
+        lg: 3,
+        xl: 4,
+      }}
+      direction={{
+        default: "column",
+        md: "row",
+      }}
+    >
+      <Button>Button 1</Button>
+      <Button>Button 2</Button>
+      <Button>Button 3</Button>
+      <Button>Button 4</Button>
+    </ResponsiveStackLayout>
+  </BreakpointsProvider>
+);
+
+export const ResponsiveChildrenExample = () => {
+  return (
+    <BreakpointsProvider>
+      <ResponsiveChildren
+        xs={<h1>XS</h1>}
+        sm={<h1>SM</h1>}
+        md={<h1>MD</h1>}
+        lg={<h1>LG</h1>}
+        xl={<h1>XL</h1>}
+      >
+        <h1>Default</h1>
+      </ResponsiveChildren>
+    </BreakpointsProvider>
+  );
+};
+
+const UseResponsiveValueExampleContent = () => {
+  const backgroundColor = useResponsiveValue({
+    default: "blue",
+    xs: "red",
+    sm: "green",
+    md: "tomato",
+    lg: "goldenrod",
+    xl: "cyan",
+  });
+
+  return (
+    <BreakpointsProvider>
```

Comment:
could the component reuse the `BreakpointContext` from `SaltProvider`?

Reply:
I think it would replace the BreakpointContext if we're happy with this solution.

Reply:
Lol just seen this after my comment